dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM centos:7
+WORKDIR /usr/src/modmail
+
+RUN yum update -y
+
+# Install NVM
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+SHELL ["bash", "-lc"]
+RUN nvm --version
```

Comment:
Is it necessary to install it on a full CentOS container? Could you use something like
node:(version here)? Here is a guide, ignore the webapp/webserver parts but it is partially applicable since this is a webserver as well as a Discord bot:
https://nodejs.org/fr/docs/guides/nodejs-docker-webapp/

Reply:
this docker file includes node version which provided at `.nvmrc` file.
we can use 'node' image when we knew what version we should use, but at this moment, we can't knew what version we should use before reading `.nvmrc` file.

so I used `centos` base image, and installed nvm to use node version that provided at `.nvmrc` file.